### PR TITLE
explain maintenance philosophy

### LINKED ISSUE #100: Improve README to explain that alpakka is a community driven effort
Improve README to explain that alpakka is a community driven effort.
We should be clear about "if you contribute, it would be nice you'd maintain it as well" etc

### PR DESCRIPTION
**Pull request by ktoso**
Resolves #100 by applying an *Old Klangian Proverb*

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,13 @@ Documentation
 
 See [reference](http://developer.lightbend.com/docs/alpakka/0.2/) and [API](http://developer.lightbend.com/docs/api/alpakka/0.2/akka/stream/alpakka/index.html) documentation pages.
 
-Contributions
--------------
+Contributions & Maintenance
```
Comment (viktorklang) [inline README.md]: Contributions & Maintainers

**Comment by ktoso** [comment]
Updated 👍

File: README.md [hunk 2]
```diff
@@ -10,10 +10,13 @@ Documentation
 
 See [reference](http://developer.lightbend.com/docs/alpakka/0.2/) and [API](http://developer.lightbend.com/docs/api/alpakka/0.2/akka/stream/alpakka/index.html) documentation pages.
 
-Contributions
--------------
+Contributions & Maintainers
+---------------------------
+
+*Alpakka does not have contributors, it only has maintainers—frequent and infrequent—and everyone helps out.*
```
Comment (drewhk) [inline README.md]: shouldn't we emphasize though that we are _looking over_ this stuff, at least at some level of supervision?

Comment (ktoso) [inline README.md]: Ok sure, do you have some specific wording in mind?

Comment (drewhk) [inline README.md]: No :)

Comment (ktoso) [inline README.md]: I added `The Akka core team keeps an eye on the project to assure its overall coherence but does not fully support these modules.` though not sure if it's good, please reword at will.

**Pull request merged**